bin/templates/project/Activity.kt

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
I'm not entirely sure if this file should make it in and it's currently not hook up to the create method. I had just created it to do some functional testing.  They claim that Kotlin is 100% interchangeable with Java as long as the stdlibs are imported.. Since that's now a thing with this change we could actually use this in favor of `Activity.java`, if you all are open to it.